test/cluster/basic_test.go

```diff
@@ -62,7 +62,7 @@ func TestBasicCluster(t *testing.T) {
 			return errors.Errorf("Redirect blocked in test client: url: '%s', referrer: '%s', #via: %d", req.URL, req.Referer(), len(via))
 		},
 	}
-	c, err := setup.NewCluster(setupConfig)
+	c, err := setup.NewCluster(setupConfig, false)
```

Comment:
Put the TLS config options (say, clusterTLSEnabled=false/true) into the the setup.Config object... it will be false by default, so all unrelated tests stay the same.